changelog_generator/repository_manager.py

```diff
@@ -60,14 +60,17 @@ def commits_since_last_tag(self) -> Sequence[Commit]:
             revision = f"{self.previous_tag}..{self.current_tag}"
         else:
             revision = self.current_tag
+        return self._get_commits(revision)
 
+    def _get_commits(self, revision: str) -> Sequence[Commit]:
```

Comment:
```suggestion
    def _get_commits(self, revision: str) -> tuple[Commit]:
```

Reply:
this repo is still in python 3.7. I could use `Tuple` I guess

Reply:
Didn't realized that, ignore it then